src/style.css

```
@@ -1,3 +1,86 @@
+* {
+  box-sizing: border-box;
+}
+
+:root {
+  --color_hover: #00acdc;
+  --color_text: #000;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+}
+
 body {
   margin: 0;
 }
+
+.logo {
+  display: block;
+  width: 40px;
+  height: 40px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav {
+  color: var(--color_hover);
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav_item {
+  margin: 0 10px;
+  text-transform: uppercase;
+  font-size: 12px;
+  line-height: 14px;
+}
+
+.nav_item:first-child {
+  margin-left: 0;
+}
+
+.nav_item:last-child {
+  margin-right: 0;
+}
```

```suggestion
  margin-right: 20px;
  text-transform: uppercase;
  font-size: 12px;
  line-height: 14px;
}

.nav_item:not(:last-child) {
  margin-right: 0;
}
```
it can help fix tests. margins should be just in one side